src/routes/book.ts

```diff
@@ -0,0 +1,58 @@
+import express, { Request, Response } from 'express';
+import { Database } from 'sqlite';
+import BookDatabase  from '../classes/BookDataClass';
+import BookFetcher from '../classes/BookFetcherClass';
+
+const router = express.Router();
+
+//TODO: setup basic auth
+//basicAuth({users: { 'Project': ' Gutenberg' }, challenge: true, unauthorizedResponse: new BasicAuthAuthorizer().getUnauthorizedResponse}),
+
+router.get('/:book_id', async (req: Request, res: Response) => {
+    const { book_id } = req.params;
```

Comment:
nit: maintain camelcase
```suggestion
    const { book_id: bookId } = req.params;
```

Reply:
This is to distinguish between a param and a variable within the scope. I get you thou.